src/disco/components/Addon/index.js

```diff
@@ -158,7 +154,7 @@ export class AddonBase extends React.Component {
     );
   }
 
-  installTheme = (event) => {
+  installTheme = (event: SyntheticEvent<any>) => {
```

Comment:
I guess we can't be any more specific than this?

Reply:
Not sure which element would make sense here. Since it is not used, I don't know..